commands/azureCommands/acr-logs-utils/tableDataManager.ts

```diff
@@ -65,7 +66,14 @@ export class LogData {
                 runListResult = await this.client.runs.list(this.resourceGroup, this.registry.name, options);
             } else {
                 runListResult = [];
-                runListResult.push(await this.client.runs.get(this.resourceGroup, this.registry.name, filter.runId));
+                // Temporal fix. Issue documented: runId must exist in the registry.
```

Comment:
:-)  You mean "temporary fix".  A temporal fix would be when you have to go back to the past in order to keep your father from being killed before you were born.

Can you point to the filed issue?  Thx.

Reply:
hahahaha. Sorry, I'm working on my grammar. It has been a while since I had to use English on a daily bases. 
I actually change this a couple of hours ago. 
The problem here is that when this function is being called from the webView when throws an error, we don't handle it properly and we need to, or else, the loading symbol stays on screen.
But when this is called from registerCommand I will still throw the error so that the function can handle it properly.

Reply:
okay.